compose/bin/start

```diff
@@ -36,29 +36,33 @@ function parseYaml {
   }'
 }
 
-# Check if volume files exist to avoid creating an empty folder
-VOLUME_LIST=$(parseYaml docker-compose.dev.yml services_app_volumes)
-IGNORE_LIST="./src/app/code ./src/m2-hotfixes ./src/patches ./src/var/log ./src/var/report ./src"
-IS_VALID=true
+if [ "$1" == "--no-dev" ]; then
+  bin/docker-compose --no-dev up -d --remove-orphans "$@"
+else
```

Comment:
Love this update! 🧡

But rather than nesting the entire remaining code within an `else` block, can we do an early exit above after the `docker-compose` line is executed in the `--no-dev` clause?

Reply:
updated